fdbt-dev/Makefile

```diff
@@ -178,6 +178,10 @@ validate-netex:
 validate-latest-file:
 	./scripts/validate_latest_netex.sh
 
+# usage is 'make file=<path-to-file> validate-this-netex'
+validate-this-netex:
```

Comment:
can we give it good name like validate-netex-file or validate-netex

Reply:
We already have one named that.